package.json

```diff
@@ -37,7 +37,6 @@
   },
   "dependencies": {
     "amd-name-resolver": "0.0.6",
-    "bower": "^1.3.12",
     "bower-config": "^1.3.0",
```

Comment:
I thought about that, and decided to tackle the biggest issue/dependency (`bower`) first. In the future we can and should make those optional too.